srcpkgs/stfl/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'stfl'
 pkgname=stfl
 version=0.24
-revision=4
+revision=5
```

Comment:
You don't need to increase the revision number if you're just updating distfiles.